modules/artifacts/checksums_intermediate.txt

```diff
@@ -1,8 +1,9 @@
 <<<<<<< Updated upstream
 =======
 >>>>>>> Stashed changes
-b10a5241504f5e02471897d95a107426b108dfd6d24d478cfc341b3dd061f1cc  target/wasm32-unknown-unknown/release/abstract_cw_staking.wasm
-db2c9fafb3808aceba41b42b07fbf43ad479c85d7ee3de199eceef7af932d500  target/wasm32-unknown-unknown/release/abstract_dca_app.wasm
-9f4af931d53aa87097b40f4f331c0185bc53d953146047dc8da4c3aef8124c97  target/wasm32-unknown-unknown/release/abstract_dex_adapter.wasm
-833a8dc2005c8c19a37b8ced7fe9b8dadeae83158d667a5daeb78c98f75418d8  target/wasm32-unknown-unknown/release/abstract_tendermint_staking_adapter.wasm
-bfd1493d7036e59c97739f951ddc55d592cf3f9a7dd8f0c4be021c625a5cd5c0  target/wasm32-unknown-unknown/release/croncat_app.wasm
```

Comment:
I think we have merge incident right above

Reply:
So don't merge this? I don't really like us committing the wasms ever time but I remember @Kayanski wanting it. Could you share with us why you wanted it here? I think it was something with the automatic artifacts folder detection